apps/hex2mif.py

```diff
@@ -0,0 +1,125 @@
+'''
+This script helps to convert hex instructions .mif files.
+To run it, use "python hex2mif.py input.txt/sv output.mif"
+
+you need to export the binary instruction as hexadecimal text file.
+But for any kind of input files, it should work fine as long as
+it uses the following format:
+DEADBEEF
+BAADF00D
+'''
+
+from asyncio.windows_events import NULL
+import sys
+
+comment_msg='''-- http://srecord.sourceforge.net/

```

Comment:
Probably a good idea to add a comment on what is this specific file that the script is running and with what "flag/offset".

Example name of sv file. (graphic_screen_test-data_mem_rv32i.sv)
and the offset (0x400/0x1000)
And the destination:
"../FPGA/mem_hex/data_mem.mif" or 
"../FPGA/mem_hex/instruction_mem.mif"